docs/how-to/evaluate/custom_metric.md

```diff
@@ -0,0 +1,7 @@
+# How to create custom Metric for Ragbits evaluation
+
+`ragbits.evaluate` package provides the implementation of metrics that measure the quality of document search pipeline within `ragbits.evaluate.metrics.document_search`
+on your data, however you are not limited to this. In order to implement custom ones for your specific use case you would need to inherit from `ragbits.evaluate.metrics.base.Metric`
+abstract class and implement `compute` method.
+
+Please find the [working example](optimize.md#define-the-metrics-and-run-the-experiment) here.
```

Comment:
I don't think linking to the optimizer is enough here, we should have a standalone example going in-detail about metrics.

Reply:
Resolved in https://github.com/deepsense-ai/ragbits/pull/217/